package.json

```diff
@@ -62,6 +62,7 @@
     "karma-phantomjs-launcher": "^1.0.0",
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "^1.7.0",
+    "less": "^2.7.1",
```

Comment:
This was an unrelated issue with npm shrinkwrap.  It wouldn't work because of an unresolved peer dependency with less (and also this: https://github.com/npm/npm/issues/6565).

In npm install, it's just a warning, but in shrinkwrap it's an error.  Hence the need to explicitly resolve it.  If there's a better way around this issue, I'd love to know...